### PR TITLE
Updated Homepage with new colours and logo

### PR DESCRIPTION
**Pull request by mattysplash**
Hey lovely team,

I've made a big change to the homepage, hoping you are happy and we can roll this out to the rest of the pages without too much pain.

I asked the Teacher about how to fit the navbar and the banner together and she said its better just to remove the banner and add more rows to the car spaces (like air bnb)  it was a bit of shame as Yumi and I spent a lot of time on the sliding banner but I think it does look more straight forward now at least and its less problems

I still want to change all the images of course, I am just not sure if we have to go back to the seed file and even the model to accomodate all that. And if that is the case then I better pull request this now before the changes get too much to review.

Screenshot 2024-04-10 at 23 12 45

### CODE REVIEW

File: app/views/pages/home.html.erb
Comment (tarakoomen) [inline app/views/pages/home.html.erb:10]: I think it would be best practice to download this image, save it to the asset folder and reference to the jpg file here. Just in case this image gets altered or deleted at the external website. Let's also fill in an alt description here for accessibility (I'm an accessibility advocate).

File: app/assets/images/spotto.png
Comment (tarakoomen) [inline app/assets/images/spotto.png:1]: Love!

File: app/views/shared/_footer.html.erb
Comment (tarakoomen) [inline app/views/shared/_footer.html.erb:1]: The footer is looking great. One question for consideration: could we get rid of a few icons, like Github and a few more? Perhaps leave Facebook, Instagram, Twitter and/or Linked-in. What do you think?

**Comment by tarakoomen** [comment]
Wow, awesome work mate!!! It's looking really good and groovy, I'm loving the style and the new logo to go with it. You raise a good question about updating the photos on individual cards. I checked the code you updated and see that there's currently not a limit (e.g., of 9 like before) of the amount of cards on the homepage, which would make it more tedious to update them individually than just the 9, I guess. Let's chat it through Thursday night and we can also ask a teacher for help. Even if we changed the image on just a few cards on the homepage, or none at all if it's too much effort, I think it's starting to look really great. Thanks for this. Happy for it to be committed. :)

**Comment by mattysplash** [comment]
Thanks, I'm unable to merge it for now. it needs at least one approval from someone else.

Good question about the 3x3 grid, I saw that there were only 9 cards displayed so i just set them in columns of 3. Looking at it again though, its not clear to me why the "each do" block is only spitting out 9 of the 20 spaces. 

Screenshot 2024-04-11 at 01 04 24

**Comment by tarakoomen** [comment]
Yes, you won't be able to merge your own work. I can't merge it either right now. I think this may be because some of my remarks need to be resolved first, such as the link to the external website photo. I think when you address those remarks, I'll be able to merge it

**Review by tarakoomen — APPROVED** [review]

**Comment by tarakoomen** [comment]
I have approved. We can implement those small changes later.

File: app/views/pages/home.html.erb
Comment (mattysplash) [reply app/views/pages/home.html.erb:10]: this image wont be used as we will swap in unique images for each space today

File: app/views/shared/_footer.html.erb
Comment (mattysplash) [reply app/views/shared/_footer.html.erb:1]: agreed I just didnt want to delete all Yumis work. but it can be a bit cleaner still

**Pull request merged**
